scripts/bootstrap.sh.in

```diff
@@ -8,7 +8,11 @@ set -u
 if test -z "$(type -p)"; then set -o pipefail; fi
 
 VERBOSE=${VERBOSE:-0}
-LOGFILE=/var/log/metalk8s-bootstrap.log
+LOGFILE=/var/log/metalk8s/bootstrap.log
+
+if [ ! -d "/var/log/metalk8s" ]; then
```

Comment:
We should probably use some `dirname` command or some variable in order to have only once the dir path?

Reply:
Also since you use `-p` not need to do a if IMO

Reply:
Minor: For me you should create this directory after the options check, just before the first `started on` message :)